packages/db/prisma/index.ts

```diff
@@ -1,23 +1,3 @@
-import { PrismaClient } from "@prisma/client";
-
-import { baseExtension, statsExtension } from "./extensions";
-
-const globalForPrisma = globalThis as unknown as { prisma: PrismaClient };
-
-const prisma_ =
-  globalForPrisma.prisma ||
-  new PrismaClient({
-    log:
-      process.env.NODE_ENV === "development"
-        ? ["query", "error", "warn"]
-        : ["error"],
-  });
-
-export const prisma = prisma_.$extends(baseExtension).$extends(statsExtension);
-
-export type BlobscanPrismaClient = typeof prisma;
-
+export * from "./client";
```

Comment:
Maybe we can do the same for the blob storage manager package. Have a client file that holds the singleton